### PR TITLE
PR: Remove locals inspection from the kernel (IPython console)

### LINKED ISSUE #21149: Spyder 6.0a1 - Pandas error crashes console
Hi - quick bug.  It seems like when you get a pandas error, like this one, it kills the Spyder iPython console:
image

### PR DESCRIPTION
**Pull request by impact27**
## Description of Changes

- Depends on https://github.com/spyder-ide/spyder-kernels/pull/464

### Issue(s) Resolved

Fixes #21149

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

### CODE REVIEW

File: spyder/config/main.py [hunk 1]
```diff
@@ -185,8 +185,6 @@
             ('debugger',
              {
               'exclude_internal': True,
-              'capture_locals': False,
-              'show_locals_on_click': False,
```
Comment (ccordoba12) [inline spyder/config/main.py:189]: This needs a bump in `CONF_VERSION` below.

File: spyder/plugins/debugger/widgets/main_widget.py [hunk 2]
```diff
@@ -292,9 +273,7 @@ def setup(self):
         # Options menu
         options_menu = self.get_options_menu()
         for item in [
```
Comment (ccordoba12) [inline spyder/plugins/debugger/widgets/main_widget.py:275]:
```suggestion
        for item in [exclude_internal_action]:
```

Leave this in a single line.

File: spyder/plugins/debugger/widgets/main_widget.py [hunk 3]
```diff
@@ -292,9 +273,7 @@ def setup(self):
...
-                exclude_internal_action,
-                capture_locals_action,
-                show_locals_on_click_action]:
+                exclude_internal_action]:
```
Comment (ccordoba12) [inline spyder/plugins/debugger/widgets/main_widget.py:276]:
```suggestion
```

**Review by ccordoba12 — COMMENTED** [review]
Some minor comments and suggestions for you @impact27, the rest looks good to me.

**Review by ccordoba12 — APPROVED** [review]
Thanks @impact27!

**Comment by mg3146** [comment]
Hi ya'll - is there any way for me to download this fix now?  Or do I have to wait until the next spyder version is released?  This bug is really killing me / my productivity... so would love to get it in somehow if possible.  Thanks again!

**Comment by ccordoba12** [comment]
Or you can wait until the next alpha, which will be released in a couple of weeks.

**Comment by mg3146** [comment]
Great, tx!

**Pull request merged**